editor-core/src/main/webapp/app/core/editorCore/model/DefaultDiagramNode.ts

```diff
@@ -99,17 +102,35 @@ export class DefaultDiagramNode implements DiagramNode {
         }
     }
 
+    setPropertyEditElementsPosition(x : number, y : number) : void {
+        let propertiesCount = 0;
+
+        for (let i in this.propertyEditElements) {
```

Comment:
You can either use here `for ... of` construction or use `i` variable also as propertiesCount